### ISSUE TITLE
If possiible, support for japanese

### ISSUE THREAD

**Issue by hetareba** [yjg30737/pyqt-openai#66]
Thank you sir for this great work.
I'm Hetareba.
I could save much time with  this tool, thanks.
If possible, could you make  japanese prompt?
yes, i can use english prompt but i use Japanse much more than English, so japanese prompt will help my work.
it is sad because i don't know how to send you some gift for this work.

**Comment by yjg30737** [comment]
No worry! will do :) By the way i did not come up with prompt translation idea. Thanks a lot.
This takes a couple of days. I will definitely tell you after that job is finished.

Gift can be great way to improve this project ! I will add paypal link or "buy me a coffee" link.
Again, thanks a lot!

**Comment by yjg30737** [comment]
OMG i have to say that i'm sorry to say this this late

"Japanese prompt" obviously you mean translating awesome-chatgpt-prompt which takes really long time.

So i will translate application's GUI like <a href="https://github.com/yjg30737/pyqt-openai#supported-languages">i mention here</a>

Also, to be honest, i translate these things most by chatgpt, but chatgpt's translation is not THAT accurate, even though it is more accurate than Google Translate.

 So i think that each one of result of translations of "awesome-chatgpt-prompt" can be not accurate as well. That's just what i'm thinking.
 
 Bottom line, I will translate application's GUI at least, but i will postpone the prompt (such as awesome-chatgpt-prompt) translation.

**Comment by yjg30737** [comment]
![image](https://github.com/yjg30737/pyqt-openai/assets/55078043/1e3881f4-ada3-4ec7-9350-9c361711f1e7)

Sorry for wait you THIS LONG.

I've been busy by company project and summer picnic and other things.

Application's translation has been completed. At least..

**Comment by hetareba** [comment]
Good morning, yjg😀.
I'm Geso.
Thank you very much for your work!
Your ChatBot is useful for using OpenAI-API.
I'll test using your NEW chatbot with awesome prompts translated into
Japanese.
Thank you, sir.

2023年8月14日(月) 8:54 yjg30737 ***@***.***>:

**Comment by yjg30737** [comment]
You're welcome !!
